src/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -160,6 +160,27 @@ const PipelineRunDetailsTab: React.FC<PipelineRunDetailsTabProps> = ({ pipelineR
                     )}
                   </DescriptionListDescription>
                 </DescriptionListGroup>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Component</DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {pipelineRun.metadata?.labels?.[PipelineRunLabel.COMPONENT] ? (
+                      pipelineRun.metadata?.labels?.[PipelineRunLabel.APPLICATION] ? (
+                        <Link
+                          to={`/stonesoup/applications/${
+                            pipelineRun.metadata.labels[PipelineRunLabel.APPLICATION]
+                          }`}
```

Comment:
Should it go to the component tab in details view?
```suggestion
                          to={`/stonesoup/applications/${
                            pipelineRun.metadata.labels[PipelineRunLabel.APPLICATION]
                          }/components`}
```

Reply:
yes it should have